Sources/MapboxDirections/Intersection.swift

```diff
@@ -111,6 +113,12 @@ public struct Intersection {
      */
     public let restStop: RestStop?
 
+    /**
+     Street class according to Mapbox Streets V8 classification.
+     
+     This value is set to `nil` of such info is not available.
+     */
+    public let streetClass: MapboxStreetClass?
```

Comment:
For consistency, move this property up next to `outletRoadClasses` and rename it to `outletDetailedRoadClass` or `outletMapboxStreetsRoadClass`, depending on the final name for the type.